static/app/utils/profiling/hooks/useProfileFilters.tsx

```diff
@@ -0,0 +1,31 @@
+import {useEffect, useState} from 'react';
+
+import {Client} from 'sentry/api';
+import {Organization, Tag} from 'sentry/types';
+import useApi from 'sentry/utils/useApi';
+import useOrganization from 'sentry/utils/useOrganization';
+
+type ProfileFilters = {[key: string]: Tag};
+
+function useProfileFilters(): ProfileFilters {
+  const api = useApi();
+  const organization = useOrganization();
+
+  const [profileFilters, setProfileFilters] = useState<ProfileFilters>({});
+
+  useEffect(() => {
+    api.clear();
```

Comment:
Actually, this will clear any in-flight requests... So we may be cancelling requests that we dont own...